tldw_Server_API/app/api/v1/endpoints/audio.py

```
@@ -2137,7 +2297,7 @@ async def _decode_and_validate_jwt_token(token: str) -> Optional[int]:
                 from tldw_Server_API.app.core.AuthNZ.quotas import increment_and_check_api_key_quota
 
                 api_mgr = await get_api_key_manager()
-                client_ip = getattr(websocket.client, "host", None)
+                client_ip = resolve_client_ip(websocket, get_settings())
```

Ruff (0.14.13)

Undefined name `resolve_client_ip`

(F821)